test/test_benchmark.py

```diff
@@ -918,6 +918,12 @@ def output_parser(output):
       return float(re.search(r'Total time: ([\d\.]+)', output).group(1))
     self.do_benchmark('memset_16mb', read_file(test_file('benchmark/benchmark_memset.cpp')), 'Total time:', output_parser=output_parser, shared_args=['-DMIN_COPY=1048576', '-DBUILD_FOR_SHELL', '-I' + test_file('benchmark')])
 
+  def test_malloc_mt(self):
```

Comment:
Maybe give the filename the same name as the test?  (it doesn't look like there is anything mimalloc specific in that file)

Reply:
This reuses the same file as in `other`. Renaming it something generic like `test_malloc_stress` would mean that the test in `other` is called `test_mimalloc` but uses a file of a different name. Unless I did something like this:
```python
  @parameterized({
    'mimalloc': (['-DWORKERS=1'],),
    'mimalloc_pthreads': (['-pthread', '-DWORKERS=4'],),
  })
  def test_malloc_stress(self, args):
    args = args + [
      '-O2',
      '-sMALLOC=mimalloc',
```
That is, add "mimalloc" in the parameterization... but that seems ugly to me. Ideas?

(The problem is that that test in `other` really just wants to test `mimalloc` - it's a multithreaded malloc stress test, and we don't want to run it on dlmalloc which is super slow on it.)

Reply:
If the test is super slow can we just speed it up so that it takes a couple of seconds on dlmalloc and few hundred milliseconds on mimalloc?    Is there some reason is needs to run for multiple seconds?

Also, if its worth stresss testing one malloc, why its it worth stress testing all them?

Reply:
I could tweak it to run faster, I guess.

But I don't think we need new coverage for dlmalloc here, I guess, so it seems odd to me to add it.

Reply:
How about we call the file something like `test_malloc_multithreading.c` ?

Reply:
Ok, renamed to that in https://github.com/emscripten-core/emscripten/pull/20651/commits/bdddfce6b90a62bc0025f049de0c66eb44fb089d

That has the downside I mentioned before in `other`, but it might be better overall, yeah.